### DISCUSSION TITLE
How to react to custom events?

### DISCUSSION THREAD

**Discussion by danielniccoli** [alpinejs/alpine#4338] [1. Help] [answered]
I'm using Bootstrap modals ([LiveDemo](https://getbootstrap.com/docs/5.3/components/modal/#live-demo)). They fire [custom events](https://getbootstrap.com/docs/5.3/components/modal/#events) like `hidden.bs.modal`. I don't get it to work to react to those events. For example:

```html
<div x-data
     @hidden.bs.modal="console.log('Modal is closed')"
     id="exampleModal"
     class="modal fade" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
    ....
</div>
``` 

I know that I can just add a script block es per the example, but I'd rather try to do it Alpine-style. Can you help?

**Comment by ekwoka** [comment, marked as answer]
https://alpinejs.dev/directives/on#dot
